login.php

```diff
@@ -127,6 +127,18 @@
     })
   })
   })
+
+  $(document).ready(function(){
```

Comment:
Move the function inside the already existing document.ready - don't need it twice